doc/3/controllers/document/create/index.md

```diff
@@ -0,0 +1,59 @@
+---
+code: true
+type: page
+title: create
+description: Creates a new document
+---
+
+# create
+
+Creates a new document in the provided index and collection.
+
+---
+
+## Arguments
+
+```java
+public CompletableFuture<ConcurrentHashMap<String, Object>> create(
+      final String index,
+      final String collection,
+      final ConcurrentHashMap<String, Object> document)
+throws NotConnectedException, InternalException
+
+public CompletableFuture<ConcurrentHashMap<String, Object>> create(
+      final String index,
+      final String collection,
+      final ConcurrentHashMap<String, Object> document,
+      final ConcurrentHashMap<String, Object> options)
+throws NotConnectedException, InternalException
+```
+
+| Arguments          | Type                                         | Description                       |
+| ------------------ | -------------------------------------------- | --------------------------------- |
+| `index`            | <pre>String</pre>                            | Index                             |
+| `collection`       | <pre>String</pre>                            | Collection                        |
+| `document`         | <pre>ConcurrentHashMap<String, Object></pre> | Content of the document to create |
```

Comment:
```suggestion
| `document`         | <pre>ConcurrentHashMap<String, Object></pre> | Document content |
```
To be consistent with [other parts](https://github.com/kuzzleio/sdk-java/pull/61/files#diff-f937e4cfa057d88243d90c6ea85244cdR38) of the documentation